Formula/qwt.rb

```diff
@@ -4,18 +4,10 @@ class Qwt < Formula
   url "https://downloads.sourceforge.net/project/qwt/qwt/6.1.3/qwt-6.1.3.tar.bz2"
   sha256 "f3ecd34e72a9a2b08422fb6c8e909ca76f4ce5fa77acad7a2883b701f4309733"
 
-  bottle do
```

Comment:
[Looks like we don't need to provide hashes](https://github.com/ogrisel/homebrew/blob/master/Library/Contributions/example-formula.rb#L94) when submitting changes to homebrew.